packages/plugin-bnb/src/actions/transfer.ts

```
@@ -0,0 +1,270 @@
+
+            // wait for the transaction to be confirmed
+            const publicClient = this.walletProvider.getPublicClient(
+                params.chain
+            );
+            await publicClient.waitForTransactionReceipt({
+                hash: resp.txHash,
+            });
+
+            return resp;
+        } catch (error) {
+            throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)